tests/trnsysGUI/storageTank/testConfigureStorageDialog.py

```diff
@@ -15,26 +17,28 @@
 class TestConfigureStorageDialog:
     storageTank = None
 
-    def triggerStorageDialog(self, qtbot):
+    @pytest.fixture()
+    def triggerStorageDialog(
+        self, qtbot, monkeypatch
+    ) -> ConfigureStorageDialog:
+        monkeypatch.setattr(QMessageBox, "exec", lambda self: QMessageBox.Ok)
+
         mainWindow = utils.createMainWindow(_PROJECT_DIR, _PROJECT_NAME)
         qtbot.addWidget(mainWindow)
         self.storageTank = utils.getDesiredTrnsysObjectFromList(
-            mainWindow.editor.trnsysObj, stw.StorageTank
-        )
-        configureStorageDialog = self.storageTank.mouseDoubleClickEvent(
-            _core.QEvent.MouseButtonDblClick, isTest=True
+            mainWindow.editor.trnsysObj, stw.StorageTank  # type: ignore
         )
-        configureStorageDialog.isTest = True
+        configureStorageDialog = self.storageTank.createStorageDialog()
         qtbot.addWidget(configureStorageDialog)
         return configureStorageDialog
 
-    def testAddHxSuccess(self, qtbot):
+    def testAddHxSuccess(self, triggerStorageDialog):
         hxInput = "90"
         hxOutput = "50"
         hxName = "Test hx"
         errors = []
 
-        configureStorageDialog = self.triggerStorageDialog(qtbot)
+        configureStorageDialog = triggerStorageDialog
```

Comment:
`triggerStorageDialog` is a very confusing name. Just name the parameter `configureStorageDialog` directly.

Reply:
@sebastian-swob: I took care of this.